optimism/src/keccak/constraints.rs

```diff
@@ -244,9 +245,10 @@ impl<Fp: Field> Constraints for KeccakEnv<Fp> {
             } // END chi
 
             // STEP iota: 4 constraints
-            for (q, c) in self.round_constants().iter().enumerate() {
+            for (q, c) in self.round_constants().to_vec().iter().enumerate() {
```

Comment:
If you're going to clone anyway (via `to_vec`), you should probably just use `into_iter` and change `c.clone()` to `c` below.

Reply:
Thanks for finding the nit, I will trace this issue here and address it asap.

https://github.com/o1-labs/proof-systems/issues/1678